Dockerfile

```diff
@@ -12,17 +11,6 @@ ENV SRC_DIR /go/src/github.com/filecoin-project/go-filecoin
 
 COPY . $SRC_DIR
 
-# Build the thing.
-RUN cd $SRC_DIR \
-&& . $HOME/.cargo/env \
-&& go run ./build/*go deps \
-&& go run ./build/*go build \
-&& go build -o ./faucet ./tools/faucet/main.go \
-&& go build -o ./genesis-file-server ./tools/genesis-file-server/main.go
-
-# Build gengen
-RUN cd
-
```

Comment:
I'm commenting here because I can't below, but on line 47 of this new file there is an `COPY --from=builder /tmp/filecoin-proof-parameters/* /tmp/filecoin-proof-parameters/` that can be removed if we are no longer generating proofs here.

Reply:
Nice catch, thanks! I'll update on Tuesday and then try testing it

Reply:
I think that this approach to not building anything in the Dockerfile results in a tight couple with Circle and will break the use-case of someone trying to build a Docker image on their machine. Not sure how important that is for the users but I think it would be useful for devs.

Reply:
@gmasgras @travisperson I agree on needing to leaving Dockerfile alone for manual builds.
Before seeing this I already backed out changes to `Dockerfile` and created a new CI specific file `Dockerfile.ci.base` based on debian stretch